onnxruntime/core/session/provider_registration.cc

```diff
@@ -67,10 +68,17 @@ ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider,
   };
 
   if (strcmp(provider_name, "QNN") == 0) {
+  // if (strcmp(provider_name, "QNN") == 0) {
```

Comment:
this comment line seems accidental?

Reply:
Yeah it was added accidentally. Removed now